database/seed.js

```diff
@@ -1,28 +1,28 @@
 const db = require('./index.js');
 const Stocks = require('./Stocks.js');
-const faker = require('faker');4
+const faker = require('faker');
 
 const generateRandomStockPrice = function() {
   return (Math.random() * (2500.00 - 0.01) + 0.01).toFixed(2);
 };
 
-const generateRandomRating = function() {
-  return (Math.random() * 99).toFixed(0);
+const generateRandomRating = () => {
+  (Math.random() * 99).toFixed(0);
 };
```

Comment:
I'm not sure that you need to use them, but faker has functions that do these exact things. It seems like your needs don't require the full scope of faker's functions so this should be fine